readthedocs/core/views/__init__.py

```diff
@@ -21,7 +21,11 @@ class NoProjectException(Exception):
     pass
 
 
-class HealthCheckView(View):
+class HealthCheckView(CDNCacheControlMixin, View):
+
+    # Never cache this view, we always want to get the live response from the server.
+    cache_response = False
```

Comment:
We didn't have a rule for this at CF, but I assume we want real time information about this

Reply:
I believe this is configured to hit the LB directly, and not go through the CDN in production, but useful to make it explicit 👍